features/traverse_history.feature

```diff
@@ -0,0 +1,50 @@
+Feature: Traverse History
```

Comment:
This might be the "right" way, but it breaks in some cases. We should either fix these cases or remove the documentation, therefore a "hidden" feature. Since we can delete hidden features.